ci/playbooks/multinode-customizations.yml

```diff
@@ -90,6 +90,7 @@
                   {% for dns_server in _crc_default_net_dns %}
                   server={{ dns_server }}
                   {% endfor %}
+                validate: "/usr/sbin/dnsmasq -C %s --test"
```

Comment:
adding link in case useful for others (haven't encountered this validate for replace before https://docs.ansible.com/ansible/latest/collections/ansible/builtin/replace_module.html#parameter-validate)

the %s is the 'temp file' created by ansible and used for the validation, before it is applied